### PR TITLE
Location name standardization

### PR DESCRIPTION
**Pull request by zzzmike**
# About the pull request

The most noticeable thing about this PR is that it converts east/west/north/south references from fuel pumps into port/starboard/etc. Which will be quite noticeable during the hijack announcements when they go online or offline. It also makes it a bit more clear on crew monitor when something is on the upper deck or lower deck, and introduces some standardization for location names. Also 
Screenshot 2023-11-20 063208  

# Explain why it's good for the game

I GET IT. It's much easier, especially for new players, if you could refer to things the way they are OOCly on your monitor. But it's also a disservice to have it both ways ingame. The crew monitor never refers to "North Lifeboat". It's starboard. PSA to anyone not good with the directional structure on crew monitor - (I'm not good with it either, don't worry) The engines are on the rear of the ship, in engineering. That's why it must be aft (or stern), and everything else must oriented from that perspective. If someone yells to get on starboard lifeboat, go north. Port? Go south.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl:
qol: Location name standardization. So, North is now Starboard. This is already how it is for everything except pumps.
/:cl:

### CODE REVIEW

File: code/game/area/almayer.dm [hunk 1]
```diff
@@ -98,23 +98,23 @@
...
-	name = "\improper Secure Storage"
+	name = "\improper Lower Deck Secure Storage"
 	icon_state = "corporatespace"
 	fake_zlevel = 2 // lowerdeck
 
 /area/almayer/command/computerlab
-	name = "\improper Computer Lab"
+	name = "\improper Lower Deck Computer Lab"
 	icon_state = "ceroom"
 	fake_zlevel = 2 // lowerdeck
```
Comment (fira) [inline code/game/area/almayer.dm:108]: Aren't these two on upper deck?

Comment (zzzmike) [reply code/game/area/almayer.dm:108]: I'm not sure, I don't think I've ever gone to either haha, I was just going off the fake zlevel thing. I will put those back anyways, thanks.

**Comment by fira** [comment]
They're the rooms the IO store intels on, top deck in the middle of the ship

**Review by fira — APPROVED** [review]

**Comment by zzzmike** [comment]
> They're the rooms the IO store intels on, top deck in the middle of the ship

Ah thanks.

**Pull request merged**